rawstockdataconsumer/src/main/java/com/jackdaw/consumer/KafkaConsumer.java

```diff
@@ -1,40 +1,41 @@
-package com.jackdaw;
+package com.jackdaw.consumer;
 
-import com.jackdaw.consumer.RawStockDataConsumerBuilder;
-import com.jackdaw.consumer.datastructures.ShiftingList;
-import com.jackdaw.consumer.datastructures.StockData;
-import com.jackdaw.consumer.thread.StockdataFileWriter;
+import com.jackdaw.consumer.model.ShiftingList;
+import com.jackdaw.consumer.model.StockData;
 import org.apache.kafka.clients.consumer.Consumer;
 import org.apache.kafka.clients.consumer.ConsumerRecords;
 
 import java.util.List;
 
-public class StockDataConsumer {
+public class KafkaConsumer {
 
-    public static void main(String[] args) throws InterruptedException {
+    public static void main(String[] args) {
         int maxSize = Integer.parseInt(args[0]);
         String topic = args[1];
         String outputPath = args[2];
-        List<StockData> stockDataList = new ShiftingList(maxSize);
-        StockdataFileWriter stockdataFileWriter = new StockdataFileWriter(stockDataList, outputPath, true);
-        Thread thread = new Thread(stockdataFileWriter);
+        ShiftingList<StockData> stockDataList = new ShiftingList<>(maxSize);
+        DataFileWriter dataFileWriter = new DataFileWriter(stockDataList, outputPath, true);
+        Thread thread = new Thread(dataFileWriter);
         thread.start();
-        runConsumer(stockDataList, topic, stockdataFileWriter);
+        runConsumer(stockDataList, topic, dataFileWriter);
     }
 
-    static void runConsumer(List<StockData> stockDataList, String topic, StockdataFileWriter stockdataFileWriter) {
-        final Consumer<Long, String> consumer = RawStockDataConsumerBuilder.createConsumer(topic);
+    private static void runConsumer(List<StockData> stockDataList, String topic, DataFileWriter dataFileWriter) {
+        final Consumer<Long, String> consumer = KafkaConsumerUtil.createConsumer(topic);
 
         final int giveUp = 100;
         int noRecordsCount = 0;
 
+        //FIXME add break to this loop
```

Comment:
There is a break to this loop, it's the one in line number 34.

Reply:
Sorry, I meant to say instead, that "continue" on line 35 is needless.